F1WM/Repositories/NewsRepository.cs

```diff
@@ -70,15 +70,82 @@ public async Task<NewsDetails> GetNewsDetails(int id)
 					news.ResultLink = new ResultLink()
 					{
 						Type = Constants.ResultTypeToLinkType[link.ResultType],
-							RaceId = (int?)(await context.Races
-								.FirstOrDefaultAsync(r => r.Date.Year == link.Year && r.Numinseason == link.Number)) ?
+						RaceId = (int?)(await context.Races
+								.FirstOrDefaultAsync(r => r.Date.Year == link.Year && r.Numinseason == link.Number))?
 							.Id
 					};
 				}
 			}
 			return news;
 		}
 
+		public async Task<IEnumerable<NewsSummary>> GetNewsByTagId(int? tagId, int page = 1, int countPerPage = 20)
```

Comment:
20 is a magic number. The default has been declared in controller already, no need to repeat it here. The comment applies to GetNewsByTypeId as well.